openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -48,4 +79,16 @@ def __init__(self, *args, **kwargs):
         data["transparency"] = self.transparency
         data["panZoom"] = self.panZoom
 
+        if preset:
+            self.log.info(
+                "Using preset: {}".format(
+                    json.dumps(preset, indent=4, sort_keys=True)
+                )
+            )
```

Comment:
Are we sure we want to be this verbose?

Reply:
Should be a debug log, but not sure whether that actually matters for performance? Are we disabling debugs calls by default?

Reply:
No, and unfortunately we can't really because the UI would still need to 'collect it' for if the user does want to see the values. Additionally, even if there was no logging handler you're still doing the json dump so best case scenario you're still having to deal with that conversion in there.

Reply:
For this plugin especially I keep on needing to see the preset cause there is quite a lot of "magic" that happens before the preset is used, so I would like to keep this. I know this is lazy developer, but got any ideas to keep it in?
Maybe the performance loss is acceptable here for now?

Reply:
If it's for development only I'd recommend putting this behind a `OPENPYPE_DEBUG` check.
I've done that in many cases, like:
```python
if os.environ.get("OPENPYPE_DEBUG") == "1":
```

I believe that value is set when running from source code.